developer/openshift/dev_setup.sh

```diff
@@ -168,6 +168,23 @@ cluster_setup() {
   | indent 2
 }
 
+# Sometimes CRDs need a few seconds to be ready
```

Comment:
I feel this should be moved to `developer/openshift/utils.sh`

Reply:
fixed